changeset:   166:42e9479fa521
tag:         tip
user:        "Daniel P. Berrange <berrange@redhat.com>"
date:        Tue May 01 11:33:16 2007 -0400
summary:     Added logic to detect RHEL5 client distro flavour

diff -r 5b37d71c88b6 -r 42e9479fa521 virtinst/DistroManager.py
--- a/virtinst/DistroManager.py	Tue May 01 11:31:32 2007 -0400
+++ b/virtinst/DistroManager.py	Tue May 01 11:33:16 2007 -0400
@@ -214,7 +214,10 @@ class RHELImageStore(RedHatImageStore):
 class RHELImageStore(RedHatImageStore):
     def isValidStore(self, fetcher, progresscb):
         if fetcher.hasFile("Server", progresscb):
-            logging.debug("Detected a RHEL 5 distro")
+            logging.debug("Detected a RHEL 5 Server distro")
+            return True
+        if fetcher.hasFile("Client", progresscb):
+            logging.debug("Detected a RHEL 5 Client distro")
             return True
         if fetcher.hasFile("RedHat", progresscb):
             logging.debug("Detected a RHEL 4 distro")

